### DISCUSSION TITLE
Google Chrome

### DISCUSSION THREAD

**Discussion by gillessad** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#850] [Q&A]
Hi, Google Chrome just added a new memory and energy safer mode, which offers two options : 
Turn on only when my battery is at 20% or lower
or
Turn on when my computer is unplugged

My question concerns the second option, with aldente activated on my mac (plugged and fully charged at 80% which is the limit that i've set) does my mac and google chrome consider it as unplugged or do they actually "see" that it is plugged ?

Thanks.

**Comment by MatthiasKerbl** [comment]
> Hi, Google Chrome just added a new memory and energy safer mode, which offers two options : Turn on only when my battery is at 20% or lower or Turn on when my computer is unplugged
> 
> My question concerns the second option, with aldente activated on my mac (plugged and fully charged at 80% which is the limit that i've set) does my mac and google chrome consider it as unplugged or do they actually "see" that it is plugged ?
> 
> Thanks.

Hi @gillessad,

When your MacBook is plugged in and charging is paused, macOS still recognizes that your MacBook is plugged in. Therefore, Google Chrome should also consider it plugged in, however, I did not test it yet.

Cheers,
Matthias
